styles.css

```diff
@@ -0,0 +1,555 @@
+  margin: 36px 16px 16px;
+  text-align: center;
+  padding: 18px 12px;
+}
+
+.aboutLogo {
+  margin-top: 24px;
+  background-color: white;
+}
+
+.aboutLogoTitle {
+  padding-top: 60px;
+  padding-bottom: 24px;
+}
+
+.aboutLogoTitleCompetition {
+  margin: 0 60px;
+}
+
+.logoImg {
+  border: solid black 2px;
+  margin: 16px 16px 0;
+}
+
+.buc {
+  background-image:
+    linear-gradient(
+      to bottom,
+      rgba(249, 52, 52, 0.5),
+      rgba(249, 52, 52, 0.5)
+    ),
+    url(./img/img1.jpg);
+  background-repeat: no-repeat;
+  width: 380px;
+  height: 210px;
+  padding: 0;
+  border: 0;
+  margin: 0;
+}
+
+.bra {
+  background-image:
+    linear-gradient(
+      to bottom,
+      rgba(249, 52, 52, 0.5),
+      rgba(249, 52, 52, 0.5)
+    ),
+    url(./img/img2.jpg);
+  background-repeat: no-repeat;
+  width: 380px;
+  height: 210px;
+  padding: 0;
+  border: 0;
+  margin: 0;
+}
+
+.cityName {
+  font-size: 34px;
+  font-weight: 900;
+}
+
+.cityNameText {
+  font-size: 12px;
+  font-weight: 900;
+}
+
+.backgroundGray {
+  background-color: #3c3f46;
+}
+
+.asa {
+  flex-direction: column;
+  align-items: center;
+  gap: 10px;
+  margin-top: 24px;
+  margin-bottom: 24px;
+}
+
+.sponsor {
+  padding: 0 16px 16px;
+}
+
+ul {
+  padding: 0;
+}
+
+.partners {
+  padding-top: 24px;
+  padding-bottom: 12px;
+  font-weight: 900;
+}
+
+.grid2 {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+}
+
+.grid3 {
+  display: none;
+}
+
+.history {
+  background-color: white;
+}
+
+.menuOpen {
+  display: block;
+  position: fixed;
+  background-color: rgb(248, 248, 248);
+  top: 0;
+  left: 0;
+  margin: 0;
+  z-index: 1;
+  width: 100vw;
+  height: 100vh;
+  padding-left: 30px;
+  padding-top: 60px;
+  font-size: 32px;
+  line-height: 2;
+  text-align: center;
+}
+
+.closeButton {
+  text-align: end;
+  padding-right: 30px;
+}
+
+@media only screen and (min-width: 768px) {
+  header {
+    background-color: #3e3e3e;
+    margin: 0;
+    padding-right: calc((100% - 766px) / 2);
+  }
+
+  nav {
+    background-color: white;
+  }
+
+  .headerItems {
+    margin: 0;
+    justify-content: end;
+  }
+
+  .headerItems > * {
+    padding: 5px 10px;
+  }
+
+  .displayNoneSmall {
+    display: block;
+  }
+
+  .displayHideBig {
+    display: none;
+  }
+
+  .menuList {
+    padding-right: calc((100% - 766px) / 2);
+    display: flex;
+    align-items: center;
+  }
+
+  .menuList > * {
+    padding: 10px 20px;
+  }
+
+  .menuBorder {
+    border: #ec5242 solid 5px;
+  }
+
+  .navLogo {
+    width: 80px;
+    margin-left: calc((100% - 766px) / 2);
+  }
+
+  .mLeft20 {
+    margin-left: calc((100% - 766px) / 2);
+  }
+
+  .mRight20 {
+    margin-right: calc((100% - 766px) / 2);
+  }
+
+  .subText {
+    margin-right: calc((100% - 766px) / 2);
+    margin-left: calc((100% - 766px) / 2);
+  }
+
+  .squares {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
+    max-width: 768px;
+    margin-left: calc((100% - 766px) / 2);
+  }
```

Comment:
- [ ] Great job 👍 . Kindly add more width to each of the squares so the text doesn't look squeezed together. This would also help your design look as close as possible to the original design 🙏 

Behance Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/190195638-bf54a2f1-4911-4569-808e-8c4dc9f727b2.png)  |  ![image](https://user-images.githubusercontent.com/74814780/190195954-20831a64-1c24-4ee2-a914-7bcd8736eba4.png)

Reply:
![Original_design](https://user-images.githubusercontent.com/108400762/190368044-82a1c9d2-46c0-4143-8840-62780ebe3555.jpg)
![MyDesign](https://user-images.githubusercontent.com/108400762/190368053-fba41925-5452-45ec-a318-41903f7cf1b8.jpg)

@Ugboaja-Uchechi  Please check the following images! It can be seen that the width of my design is slightly bigger so the width is no issue in this case. The "issue" is that there is too much content and the container gets stretched. [width is measured in pixels for the page width of 1030px (matching the original design)]